xml/System/IDisposable.xml

```diff
@@ -50,7 +50,7 @@
  For a detailed discussion about how this interface and the <xref:System.Object.Finalize%2A?displayProperty=fullName> method are used, see the [Garbage Collection](http://msdn.microsoft.com/library/22b6cb97-0c80-4eeb-a2cf-5ed7655e37f9) and [Implementing a Dispose Method](~/docs/standard/garbage-collection/implementing-dispose.md) topics.  
   
 ## Using an object that implements IDisposable  
- Implement <xref:System.IDisposable> only if you are using unmanaged resources directly. If your app simply uses an object that implements <xref:System.IDisposable>, don't provide an <xref:System.IDisposable> implementation. Instead, you should call the object's <xref:System.IDisposable.Dispose%2A?displayProperty=fullName> implementation when you are finished using it. Depending on your programming language, you can do this in one of two ways:  
+ You should call the object's <xref:System.IDisposable.Dispose%2A?displayProperty=fullName> implementation when you are finished using it. Depending on your programming language, you can do this in one of two ways:  
```

Comment:
@alfredmyers, the first sentence is certainly a duplicate and should be removed. But the beginning of the second sentence isn't a duplicate and provides context. Could you add back a condensed version of it, so that the paragraph's opening sentence now reads, "If your app simply uses an object that implements the IDisposable interface, you should call the object's Dispose implementation..."?

Reply:
I find it kind of redundant given the text is immediately under the section header "Using an object that implements IDisposable" but I don't mind making the change if you think it is really necessary.

Reply:
@alfredmyers, I agree that it's somewhat redundant, but at the same time, the redundancy is useful, since it creates a paragraph that stands alone without the header.